paasta_tools/api/views/instance.py

```diff
@@ -207,8 +207,18 @@ def kubernetes_job_status(
 ) -> None:
     app_id = job_config.get_sanitised_deployment_name()
     kstatus["app_id"] = app_id
+    kstatus["pods"] = []
     if verbose > 0:
-        kstatus["slaves"] = [pod.spec.node_name for pod in pod_list]
+        for pod in pod_list:
+            kstatus["pods"].append(
+                {
+                    "name": pod.metadata.name,
```

Comment:
we probably need the namespace as well?

Reply:
Right. I forgot that not everything is running under paasta namespace. Added it.